documentation/library_guide/cmake_support.rst

```diff
@@ -0,0 +1,108 @@
+
+Some useful CMake variables (`here <https://cmake.org/cmake/help/latest/manual/cmake-variables.7.html>`_ you can find a full list of CMake variables for the latest version):
+
+- `CMAKE_CXX_COMPILER <https://cmake.org/cmake/help/latest/variable/CMAKE_LANG_COMPILER.html>`_ - C++ compiler used for build, e.g. ``CMAKE_CXX_COMPILER=dpcpp``.
+- `CMAKE_BUILD_TYPE <https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html>`_ - build type that affects optimization level and debug options, values: ``RelWithDebInfo``, ``Debug``, ``Release``, ...; e.g. ``CMAKE_BUILD_TYPE=RelWithDebInfo``.
+- `CMAKE_CXX_STANDARD <https://cmake.org/cmake/help/latest/variable/CMAKE_CXX_STANDARD.html>`_ - C++ standard, e.g. ``CMAKE_CXX_STANDARD=17``.
+
+Requirements
+============
+The minimal supported CMake version for |onedpl_short| is 3.11 on Linux and 3.20 on Windows.
+
+The supported `CMake Generator <https://cmake.org/cmake/help/latest/manual/cmake-generators.7.html#ninja-generators>`_ for Linux is `Unix Makefiles <https://cmake.org/cmake/help/latest/generator/Unix%20Makefiles.html>`_ (default).  In the Windows environment, the supported generator is `Ninja <https://cmake.org/cmake/help/latest/generator/Ninja.html>`_ as described in the `Intel® oneAPI DPC++/C++ Compiler Developer Guide and Reference <https://www.intel.com/content/www/us/en/docs/dpcpp-cpp-compiler/developer-guide-reference/current/use-cmake-with-the-compiler.html>`_ which may be specified via ``-GNinja``.
+
+|onedpl_short| Backend Options
+==============
+
+Backend for Parallel Execution Policies (par and par_unseq)
+-----------------------------------------------------------
+The |onedpl_short| backend for parallel execution policies controls how algorithms with parallel execution policies (``par`` or ``par_unseq``) are implemented.  This option is controlled via the ``ONEDPL_PAR_BACKEND`` setting.
+
++--------------------+-----+--------+--------+
+| ONEDPL_PAR_BACKEND | TBB | OpenMP | Serial |
++====================+=====+========+========+
+| [not set]          |   oneDPL heuristics   |
++--------------------+-----+--------+--------+
+| tbb                |  X  |        |        |
++--------------------+-----+--------+--------+
+| openmp             |     |    X   |        |
++--------------------+-----+--------+--------+
+| serial             |     |        |    X   |
++--------------------+-----+--------+--------+
+
+The |onedpl_short| heuristics are the following: the first suitable backend is chosen among ``oneTBB``, ``OpenMP`` and ``Serial``, in that order.  If ``ONEDPL_PAR_BACKEND`` is specified, but the selected backend is not found or unsupported, |onedpl_short| is considered not found (``oneDPL_FOUND=False``).
+
+Backend for Device Execution Policies
+-----------------------------------------------------------
+The |onedpl_short| backend for device execution policies controls if device policies are enabled.
+
++-------------------+
+|       DPCPP       |
++-------------------+
```

Comment:
```suggestion
+====================+
```
DPCPP will be bold with that change as other backend in the table above.

Reply:
done.